client/asset/btc/btc.go

```diff
@@ -1809,11 +1809,11 @@ func fund(utxos []*compositeUTXO, enough func(uint64, uint64) bool) (
 // sent (true) or if the original coins were returned unmodified (false).
 //
 // A split transaction nets additional network bytes consisting of
-// - overhead from 1 transaction
-// - 1 extra signed p2wpkh-spending input. The split tx has the fundingCoins as
-//   inputs now, but we'll add the input that spends the sized coin that will go
-//   into the first swap if the split tx does not add excess baggage
-// - 2 additional p2wpkh outputs for the split tx sized output and change
+//   - overhead from 1 transaction
+//   - 1 extra signed p2wpkh-spending input. The split tx has the fundingCoins as
+//     inputs now, but we'll add the input that spends the sized coin that will go
+//     into the first swap if the split tx does not add excess baggage
+//   - 2 additional p2wpkh outputs for the split tx sized output and change
```

Comment:
Hmm, I must have done go fmt with Go 1.19 rc1 which  has new formatting.  https://tip.golang.org/doc/go1.19#go-doc

Since older gofmt versions don't try to change it back, I think I'll leave it.